Platform/CommonBoardPkg/CfgData/CfgData_Tcc.yaml

```diff
@@ -41,8 +41,16 @@
                      Enable will allocate 1 way of LLC; if Cache Configuration subregion is available, it will allocate based on the subregion.
       length       : 0x1
       value        : 0x0
-  - Dummy        :
+  - TccErrorLogEn    :
+      name         : TCC Error logging Enable
+      type         : Combo
+      option       : 0:Disabled, 1:Enabled
+      help         : >
+                     Enable/Disable Error logging for Tcc flow.
       length       : 0x1
       value        : 0x0
+  - Dummy        :
+      length       : 0x0
+      value        : 0x0
 

```

Comment:
Dummy could be removed since its length is 0

Reply:
Done